### PR TITLE
 permissions.yml maui

### PR DESCRIPTION
**Pull request by adambash**
In The Trailer when he was telling Moana His Life it showed he jumped right next to the sun and with his magic fishing hook he pulled an island out of the sea. That Explains his haste and jump. If You Don't Believe Me Watch The Trailer It Shows Over there. He Also Has ShapeShifting Abilities Cause He Can Turn Into Animals In The Movie And Trailer.

### CODE REVIEW

**Comment by malonnnn** [comment]
the maui: node is not spaced correctly

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -998,6 +998,41 @@ groups:
...
+        - ch.alias.run2
+        - ch.alias.haste0
+        - ch.alias.haste1
+        - ch.alias.haste2
+        - ch.alias.haste3
+        - ch.alias.haste4
+        - ch.alias.haste5
+        - ch.alias.haste6
+        - ch.alias.haste7
+        - ch.alias.haste8
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: need evidence of how strong. if no evidence, then haste 5

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -2515,7 +2550,8 @@ groups:
         - ch.alias.haste5
         - ch.alias.haste6
         - ch.alias.haste7
-  supermanprimeonemillion:
+  
+  primeonemillion:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: why did u change this?

**Comment by malonnnn** [comment]
include details of what hero in the title of the PR

File: PermissionsEx/permissions.yml [hunk 3]
```diff
@@ -998,6 +998,41 @@ groups:
...
+    options:
+      prefix: '&3M&fa&6u&9i'
+      suffix: '&3'
+    inheritance:
+    - mortal
+    worlds:
+      survival:
+        permissions:
+        - essentials.fly
+        - essentials.speed.fly
```
Comment (adambash) [inline PermissionsEx/permissions.yml]: He Can Turn Into An Eagle Cause He Is A ShapeShifter. I Dont Need To Give A website it's on the commercial

File: PermissionsEx/permissions.yml [hunk 4]
```diff
@@ -998,6 +998,58 @@ groups:
...
+      maui:
+    options:
+      prefix: '&3M&fa&6u&9i'
+      suffix: '&3'
+    inheritance:
+    - mortal
+    worlds:
+      survival:
+        permissions:
+        - libsdisguises.disguise
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: disguising
the trailer did not show this

File: PermissionsEx/permissions.yml [hunk 5]
```diff
@@ -998,6 +998,58 @@ groups:
...
+        - libsdisguises.disguise.mushroom_cow.-setinvisible
+        - libsdisguises.disguise.ocelot.-setinvisible
+        - libsdisguises.disguise.pig.-setinvisible
+        - libsdisguises.disguise.rabbit.-setinvisible
+        - libsdisguises.disguise.sheep.-setinvisible
+        - libsdisguises.disguise.silverfish.-setinvisible
+        - libsdisguises.disguise.spider.-setinvisible
+        - libsdisguises.disguise.squid.-setinvisible
+        - libsdisguises.disguise.wolf.-setinvisible
+        - essentials.fly
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: flight
the trailer did not show this

File: PermissionsEx/permissions.yml [hunk 6]
```diff
@@ -2515,6 +2568,7 @@ groups:
         - ch.alias.haste5
         - ch.alias.haste6
         - ch.alias.haste7
+
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: delete this empty line

File: PermissionsEx/permissions.yml [hunk 7]
```diff
@@ -7569,7 +7623,7 @@ users:
   df9ce472-5cc0-425d-925b-ea95d9493cb0:
     options:
       name: PotatoLlama123
-  2b684873-2571-4707-8347-ec87f6181199:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: why did you delete this

File: PermissionsEx/permissions.yml [hunk 8]
```diff
@@ -2515,7 +2569,7 @@ groups:
         - ch.alias.haste5
         - ch.alias.haste6
         - ch.alias.haste7
-  supermanprimeonemillion:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: why are there spaces added here

Comment (adambash) [inline PermissionsEx/permissions.yml]: oops i will fix

File: PermissionsEx/permissions.yml (hunk 7 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: oops i will fix

File: PermissionsEx/permissions.yml (hunk 1 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: http://www.metatube.com/en/videos/309339/Moana-US-Teaser-Trailer/

File: PermissionsEx/permissions.yml (hunk 4 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: http://www.metatube.com/en/videos/309339/Moana-US-Teaser-Trailer/

File: PermissionsEx/permissions.yml (hunk 5 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: http://www.metatube.com/en/videos/309339/Moana-US-Teaser-Trailer/

File: PermissionsEx/permissions.yml [hunk 9]
```diff
@@ -7569,7 +7623,7 @@ users:
   df9ce472-5cc0-425d-925b-ea95d9493cb0:
     options:
       name: PotatoLlama123
-  2b684873-2571-4707-8347-ec87f6181199:
+  2b684873--4707-8347-ec87f6181199:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: Why did you delete that part?

File: PermissionsEx/permissions.yml (hunk 8 above)
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: still waiting for u to fix

File: PermissionsEx/permissions.yml [hunk 10]
```diff
@@ -998,7 +998,61 @@ groups:
         - ch.alias.haste3
         - ch.alias.carryplayer
         - ch.alias.dismountplayer
-  darkseid:
+      maui:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: this is still spaced incorrectly

File: PermissionsEx/permissions.yml (hunk 9 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: oops

File: PermissionsEx/permissions.yml [hunk 11]
```diff
@@ -998,7 +998,61 @@ groups:
         - ch.alias.haste3
         - ch.alias.carryplayer
         - ch.alias.dismountplayer
-  darkseid:
+     maui:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: This is STILL spaced incorrectly

**Comment by malonnnn** [comment]
accepted - merged by hand
